inc/plugins/class-dashboard.php

```diff
@@ -221,17 +221,44 @@ public function enqueue_options_assets() {
 			apply_filters(
 				'otter_dashboard_data',
 				array(
-					'version'            => OTTER_BLOCKS_VERSION,
-					'assetsPath'         => OTTER_BLOCKS_URL . 'assets/',
-					'stylesExist'        => is_dir( $basedir ) || boolval( get_transient( 'otter_animations_parsed' ) ),
-					'hasPro'             => Pro::is_pro_installed(),
-					'upgradeLink'        => tsdk_utmify( Pro::get_url(), 'options', Pro::get_reference() ),
-					'docsLink'           => Pro::get_docs_url(),
-					'showFeedbackNotice' => $this->should_show_feedback_notice(),
-					'deal'               => ! Pro::is_pro_installed() ? $offer->get_localized_data() : array(),
-					'hasOnboarding'      => false !== get_theme_support( FSE_Onboarding::SUPPORT_KEY ),
-					'days_since_install' => round( ( time() - get_option( 'otter_blocks_install', time() ) ) / DAY_IN_SECONDS ),
-					'rootUrl'            => get_site_url(),
+					'version'                => OTTER_BLOCKS_VERSION,
+					'assetsPath'             => OTTER_BLOCKS_URL . 'assets/',
+					'stylesExist'            => is_dir( $basedir ) || boolval( get_transient( 'otter_animations_parsed' ) ),
+					'hasPro'                 => Pro::is_pro_installed(),
+					'upgradeLink'            => tsdk_utmify( Pro::get_url(), 'options', Pro::get_reference() ),
+					'docsLink'               => Pro::get_docs_url(),
+					'showFeedbackNotice'     => $this->should_show_feedback_notice(),
+					'deal'                   => ! Pro::is_pro_installed() ? $offer->get_localized_data() : array(),
+					'hasOnboarding'          => false !== get_theme_support( FSE_Onboarding::SUPPORT_KEY ),
+					'days_since_install'     => round( ( time() - get_option( 'otter_blocks_install', time() ) ) / DAY_IN_SECONDS ),
+					'rootUrl'                => get_site_url(),
+					'neveThemePreviewUrl'    => esc_url(
+						add_query_arg(
+							array(
+								'theme' => 'neve',
+							),
+							admin_url( 'theme-install.php' )
+						)
+					),
+					'neveThemeActivationUrl' => esc_url(
+						add_query_arg(
+							array(
+								'action'     => 'activate',
+								'stylesheet' => 'neve',
+								'_wpnonce'   => wp_create_nonce( 'switch-theme_neve' ),
+							),
+							admin_url( 'themes.php' )
+						)
+					),
+					'neveDashboardUrl'       => esc_url(
+						add_query_arg(
+							array(
+								'page' => 'neve-welcome',
+							),
+							admin_url( 'admin.php' )
+						)
+					),
+					'neveInstalled'          => 'neve' === get_template(),
```

Comment:
Just one thing, we can use `NEVE_VERSION` constant for the check.

Reply:
Done! Thank you for review